openfl-tutorials/interactive_api/PyTorch_TinyImageNet/workspace/pytorch_tinyimagenet.ipynb

```diff
@@ -296,13 +297,15 @@
     "\n",
     "class Net(nn.Module):\n",
     "    def __init__(self):\n",
+    "        torch.manual_seed(0)\n",
     "        super(Net, self).__init__()\n",
     "        self.model = torchvision.models.mobilenet_v2(pretrained=True)\n",
     "        self.model.requires_grad_(False)\n",
     "        self.model.classifier[1] = torch.nn.Linear(in_features=1280, \\\n",
     "                        out_features=200, bias=True)\n",
     "\n",
     "    def forward(self, x):\n",
+    "        torch.manual_seed(0)\n",
```

Comment:
I believe we discussed that this should be fixed, since fixing a random seed inside `forward()` leads to multiple seed fixes during each `forward()` call